smac/facade/smac_facade.py

```diff
@@ -111,7 +113,13 @@ def __init__(self,
                                    scenario.cs)
 
         # initialize tae_runner
-        if tae_runner is None:
+        if tae_runner is not None and callable(tae_runner):
+            tae_runner = ExecuteTAFunc(ta=tae_runner,
+                                       stats=self.stats,
+                                       run_obj=scenario.run_obj,
+                                       runhistory=runhistory,
+                                       par_factor=scenario.par_factor)
+        elif tae_runner is None:
```

Comment:
the exception should also mention that a passed None will lead to a TAE based on the call in the scenario.